openmsistream/kafka_wrapper/consumer_and_producer_group.py

```diff
@@ -136,11 +166,13 @@ def get_init_args_kwargs(cls, parsed_args):
         args = [
             *superargs,
             parsed_args.config,
-            parsed_args.consumer_topic_name,
```

Comment:
I believe this is the source of the failing test.

Reply:
@Xarthisius : yep, that is the cause of the error. But it cannot be put back in ConsumerAndProducer group now that it might be used for producing only, no consuming. So I fixed this in the place where it becomes required. While there, noticed that the documentation of how `topic_name`, `consumer_topic_name`, and `producer_topic_name` works was not commensurate with the implementation (in particular: if `topic_name` is set, it should be used as the default for the other two). Fixing that now.

Reply:
@Xarthisius : hmm, that was definitely the issue with `DataFileDownloadDirectory`, but now I'm getting an error with the `DataFileUploadDirectory` CI tests, and I don't understand because it works when I run it locally....

Reply:
@Xarthisius : thanks for the help! This has been fixed.